components/facebook_groups/facebook_groups.app.mjs

```diff
@@ -7,7 +10,7 @@ export default {
     group: {
       type: "string",
       label: "Group",
-      description: "Identifier of a group",
+      description: "Identifier of a group. The Pipedream app must be installed in the Facebook group in order to make any API calls or receive data.",
       async options() {
         const userId = await this.getUserId();
         const opts = {
```

Comment:
You could add a try/catch block in the `group` async options, just like in the `post` async options. Both of the sources have a `group` prop, so if they received an error, the user would see the error message & link while setting up the props, and wouldn't be able to deploy.

Reply:
I think the issue with this is that the groups async option which uses `listGroups` wouldn't actually fail if the Pipedream app wasn't installed in the group, so it differs as compared to the `post` async option where `listPosts` actually requires the prior installation. 

I will merge this PR for now and come back to error message for sources, but I think that having the similar try/catch block on `groups` would not actually flag the lack of installation of the app given what the `groups` async options actually calls.